pkg/operator/logging/logging.go

```diff
@@ -131,12 +126,8 @@ func loggerFromFile(ctx context.Context, component string) *zap.SugaredLogger {
 // ConfigureGlobalLoggers sets up any package-wide loggers like "log" or "klog" that are utilized by other packages
 // to use the configured *zap.SugaredLogger from the context
 func ConfigureGlobalLoggers(ctx context.Context) {
```

Comment:
Do we even need this function anymore or can we just directly call these functions in `operator.go`?

Reply:
do'h, absolutely. Blind me :)